core/block_svg.ts

```diff
@@ -191,7 +162,7 @@ export class BlockSvg extends Block implements IASTNodeLocationSvg,
         workspace.getRenderer().makePathObject(this.svgGroup_, this.style);
 
     const svgPath = this.pathObject.svgPath;
-    (svgPath as AnyDuringMigration).tooltip = this;
+    (svgPath as any).tooltip = this;
```

Comment:
Is this definitely how we want to handle this?

Reply:
We've done this in a couple other places; AFAICT it's our best option without refactoring how tooltips work - do you know of any alternatives that might be better?

Reply:
haha nope, just wanted to check.